### ISSUE TITLE
Crash when x-ray dashboard doesn't contain parameters

### ISSUE THREAD

**Issue by kulyk** [metabase/metabase#19921]
**Describe the bug**
When x-raying, if the resulting dashboard doesn't have parameters, the FE crashes.

**Logs**
```
parameter-values.js:171 Uncaught TypeError: Cannot read properties of null (reading 'map')
    at getParameterValuesBySlug (parameter-values.js:171:1)
    at DashCard.render (DashCard.jsx:159:1)
    at finishClassComponent (react-dom.development.js:17160:1)
    at updateClassComponent (react-dom.development.js:17110:1)
    at beginWork (react-dom.development.js:18620:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:188:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:237:1)
    at invokeGuardedCallback (react-dom.development.js:292:1)
    at beginWork$1 (react-dom.development.js:23203:1)
    at performUnitOfWork (react-dom.development.js:22154:1)
```

**To Reproduce**

1. Create a SQL question: `SELECT * FROM ORDERS`
2. Create a GUI query based on the native question: summarize Count by Created At
3. Click any dot on the line chart visualization
4. Click "x-ray" in a popover
5. FE crashes

**Expected behavior**
It should be possible to x-ray the data and see the dashboard

**Demo**

https://user-images.githubusercontent.com/17258145/151020828-4d91f3fb-26e9-4cf4-b94b-308eb2aa85a8.mov

**Information about your Metabase Installation:**

`master` branch on [commit](https://github.com/metabase/metabase/commit/7a81a9e59952257c2946bfc5485b0a41ccb701da)

**Comment by flamber** [comment]
I don't think it's a regression. Rather just a slight change in behavior perhaps compared to previously #15655

**Comment by kulyk** [comment]
Thanks, @flamber! I have a fix for this issue, let me check if #15655 is still an issue, there are a lot of improvements in the nested queries land

**Comment by kulyk** [comment]
Yeah, seems like it's the same thing as #15655. Closing this one and will focus on the original issue
